### ISSUE TITLE
Make image alts a requirement

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/pinafore#514]
Prior art: "Better Mastodon" https://toot.rodeo/@rmlewisuk/100629111281920185

**Comment by nolanlawson** [comment]
At the very least, add an opt-in option to do this.

**Comment by NickColley** [comment]
We could do an opt-out 'reminder' function, some people cannot write alt-text because of their own disability so we should never force everyone to add them.

**Comment by NickColley** [comment]
Consolidating into https://github.com/nolanlawson/pinafore/issues/2250
